packages/bladebones/stubs/defaults/routes/web.stub

```diff
@@ -32,73 +32,59 @@ Route::get('/', function () {
     return view('welcome');
 });
 
-Route::middleware('auth')->group(function () {
-    Route::get('/home', fn () => view('home'));
-});
-
-Route::prefix('/auth')->group(function () {
-    Route::middleware('guest')->group(function () {
-        Route::get('/register', [RegisterController::class, 'create'])->name('register');
-        Route::post('/register', [RegisterController::class, 'store']);
-        Route::delete('/register', [RegisterController::class, 'destroy']);
-
-        Route::get('/login', [LoginController::class, 'create'])->name('login');
-        Route::post('/login', [LoginController::class, 'store']);
+Route::middleware('guest')->group(function () {
+    Route::get('/auth/login', [LoginController::class, 'create'])->name('login');
+    Route::post('/auth/login', [LoginController::class, 'store']);
 
-        Route::prefix('/recover-account')->group(function () {
-            Route::get('/', [AccountRecoveryRequestController::class, 'create'])->name('recover-account');
-            Route::post('/', [AccountRecoveryRequestController::class, 'store']);
-
-            Route::get('/{token}', [AccountRecoveryChallengeController::class, 'create'])->name('recover-account.challenge');
-            Route::post('/{token}', [AccountRecoveryChallengeController::class, 'store']);
-        });
-
-        Route::prefix('/login/challenge')->middleware(EnsurePreAuthenticated::class)->group(function () {
-            Route::get('/', [MultiFactorChallengeController::class, 'create'])->name('login.challenge.multi_factor');
-            Route::post('/', [MultiFactorChallengeController::class, 'store']);
-        });
+    Route::middleware(EnsurePreAuthenticated::class)->group(function () {
+        Route::get('/auth/login/challenge', [MultiFactorChallengeController::class, 'create'])->name('login.challenge.multi_factor');
```

Comment:
```suggestion
        Route::get('/auth/login/challenge', [MultiFactorChallengeController::class, 'create'])->name('login.challenge');
```

Reply:
Just an idea, because I would otherwise think that there is another route `login.challenge`